runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -3735,6 +3735,11 @@ J9::Z::CodeGenerator::suppressInliningOfRecognizedMethod(TR::RecognizedMethod me
    if (self()->isMethodInAtomicLongGroup(method))
       return true;
 
+   if (method == TR::sun_nio_cs_SingleByteDecoder_decodeToLatin1Impl)
```

Comment:
We should also check that acceleration of the intrinsic is enabled or not - before suppressing the inlining.

Reply:
Added option to toggle acceleration here: bda4b2721e47d238dfdc5bb9c45c9247ac57feb0